src/Microsoft.PowerShell.Commands.Utility/commands/utility/ImportPowerShellDataFile.cs

```diff
@@ -0,0 +1,102 @@
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Management.Automation;
+using System.Management.Automation.Language;
+
+namespace Microsoft.PowerShell.Commands
+{
+    /// <summary>
+    /// This class implements Import-PowerShellDataFile command.
+    /// </summary>
+    [Cmdlet(VerbsData.Import, "PowerShellDataFile", DefaultParameterSetName = "ByPath",
+        HelpUri = "https://go.microsoft.com/fwlink/?LinkID=623621", RemotingCapability = RemotingCapability.None)]
+    public class ImportPowerShellDataFileCommand : PSCmdlet
+    {
+        bool _isLiteralPath;
+
+        /// <summary>
+        /// Path specified, using globbing to resolve
+        /// </summary>
+        [Parameter(Mandatory = true,
+            Position = 0,
+            ParameterSetName = "ByPath",
+            ValueFromPipeline = true,
+            ValueFromPipelineByPropertyName = true)]
+        [Alias("PSPath")]
+        [ValidateNotNullOrEmpty()]
+        [SuppressMessage("Microsoft.Performance", "CA1819:PropertiesShouldNotReturnArrays")]
+        public string[] Path { get; set; }
+
+        /// <summary>
+        /// Specifies a path to one or more locations, without globbing
+        /// </summary>
+        [Parameter(Mandatory = true,
+            Position = 0,
+            ParameterSetName = "ByLiteralPath")]
+        [ValidateNotNullOrEmpty()]
+        [SuppressMessage("Microsoft.Performance", "CA1819:PropertiesShouldNotReturnArrays")]
+        public string[] LiteralPath
+        {
+            get { return Path; }
+            set { _isLiteralPath = true; Path = value; }
```

Comment:
A quick search gives that the pattern is in
```
AclCommands.cs
CertificateCommands.cs
CSVCommands.cs
ExportAliasCommand.cs
MatchString.cs
Out-File.cs
SaveHelpCommand.cs
SignatureCommands.cs
StartTranscriptCmdlet.cs
UpdateHelpCommand.cs
WriteFormatDataCommand.cs
XmlCommands.cs
```
But the cost of changing is very small.